File: C++/program122.cpp
#include<iostream>
using namespace std;

void Display(int ptr[], int iSize)
{
    cout<<"Elements of array "<<'\n';
    int iCnt = 0;

    for(iCnt = 0; iCnt < iSize; iCnt++)
    {
        cout<<ptr[iCnt]<<'\n';
    }

}
int main()
{
    int Arr[5] = { '\0'};
    int iCnt = 0;

    cout<<"Enter array elements "<<'\n';
    for(iCnt = 0; iCnt < 5; iCnt++)
    {
        cin>>Arr[iCnt];
    }
    
    Display(Arr,5); // Arr = 100(base address)
    return 0;
}

File: program122.java
import java.util.*;

class ArrayProgram
{
    public void Display(int [] Arr, int iSize)
    {
        int iCnt = 0;
        System.out.println("Array elements are :");

        for(iCnt = 0; iCnt < iSize; iCnt++)
        {
            System.out.println(Arr[iCnt]);
        }


        
    }
}

class program122
{
    public static void main(String A[])
    {
       
        int [] Arr = new int [5];
        int iCnt = 0;
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter elements :");

        for(iCnt = 0; iCnt < 5 ; iCnt++)
        {
            Arr[iCnt] = sobj.nextInt() ;
        }

        ArrayProgram aobj = new ArrayProgram();
        aobj.Display(Arr,5);

       

    }
}